src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
+    // Their version appended the packageName and "Event" to
+    // "sysInstallPackage", but I have revised it to pass the packageName as a
+    // separate argument.
+    // Due to the range of circumstances in which this method can be called
+    // I have also included:
+    // * the invoking "module" argument in case it is necesary to differentiate
+    // the way in which the installPackage code is called - I think this could
+    // be a factor in some use cases that Zoilder would not have experienced in
+    // their usage.
+    // * the original package filename
+    //
+    // The originators comment was (ES):
+    // Se ejecuta el evento que indica que se ha instalado el paquete.
+    // Así el propio paquete puede auto-inicializarse sin tener que reabrir la
+    // sesión.
+    // which I think translates to (EN):
+    // The event that indicates that the package is installed runs.
+    // Thus the package itself can auto-initialize without having to reopen the
+    // session.
+    TEvent installEvent;
+    installEvent.mArgumentList.append( tr( "sysInstallPackageEvent", "This string may not want/need to be translated" ) );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( packageName );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( QString::number( module ) );
```

Comment:
Well, English words instead of numbers are much more readable - and serve partly as comment at the same time. Everyone writing any code is now familiar with English words enough.
All numbers will require commenting their meaning in all scripts later if authors want them to be readable to anyone else.
Also, string constants should NOT be translated or most scripts will stop working once Mudlet runs with different locale, which is stupid.